linux_v20.yml

```diff
@@ -0,0 +1,80 @@
+name: tcrm
+channels:
+- defaults
+dependencies:
+- backports=1.0=py27_0
+- backports_abc=0.5=py27_0
+- basemap=1.0.7=np111py27_0
+- cairo=1.14.8=0
+- curl=7.49.0=1
+- cycler=0.10.0=py27_0
+- dbus=1.10.10=0
+- expat=2.1.0=0
+- fontconfig=2.12.1=3
+- freetype=2.5.5=2
+- functools32=3.2.3.2=py27_0
+- geos=3.4.2=0
+- glib=2.50.2=1
+- gst-plugins-base=1.8.0=0
+- gstreamer=1.8.0=0
+- harfbuzz=0.9.39=2
+- hdf4=4.2.12=1
+- hdf5=1.8.17=1
+- icu=54.1=0
+- jpeg=9b=0
+- libffi=3.2.1=1
+- libgcc=5.2.0=0
+- libgfortran=3.0.0=1
+- libiconv=1.14=0
+- libnetcdf=4.4.1=1
+- libpng=1.6.27=0
+- libxcb=1.12=1
+- libxml2=2.9.4=0
+- matplotlib=1.5.1=np111py27_0
+- mkl=2017.0.1=0
+- netcdf4=1.2.4=np111py27_0
+- numpy=1.11.3=py27_0
+- openssl=1.0.2l=0
+- pandas=0.20.1=np111py27_0
+- pango=1.40.3=1
+- patsy=0.4.1=py27_0
+- pcre=8.39=1
+- pip=9.0.1=py27_1
+- pixman=0.34.0=0
+- pycairo=1.10.0=py27_0
+- pyparsing=2.1.4=py27_0
+- pyqt=4.11.4=py27_4
+- python=2.7.13=0
+- python-dateutil=2.6.0=py27_0
+- pytz=2017.2=py27_0
+- qt=4.8.7=4
+- readline=6.2=2
+- scipy=0.19.0=np111py27_0
+- seaborn=0.7.1=py27_0
+- setuptools=27.2.0=py27_0
+- shapely=1.5.16=py27_0
+- simplejson=3.10.0=py27_0
+- singledispatch=3.4.0.3=py27_0
+- sip=4.18=py27_0
+- six=1.10.0=py27_0
+- sqlite=3.13.0=0
+- ssl_match_hostname=3.4.0.2=py27_1
+- statsmodels=0.8.0=np111py27_0
+- subprocess32=3.2.7=py27_0
+- tk=8.5.18=0
+- tornado=4.5.1=py27_0
+- wheel=0.29.0=py27_0
+- zlib=1.2.8=3
+- pip:
+  - backports-abc==0.5
+  - backports.ssl-match-hostname==3.4.0.2
+  - click==6.7
+  - colormap==1.0.1
+  - flask==0.12.2
+  - itsdangerous==0.24
+  - jinja2==2.9.6
+  - markupsafe==1.0
+  - redis==2.10.5
+  - werkzeug==0.12.2
+prefix: /home/u53337/miniconda2/envs/tcrm
```

Comment:
Can this last line be removed? I'm wary of anything that has a fixed path in configurations